snowflake/src/test/java/org/apache/iceberg/snowflake/JdbcSnowflakeClientTest.java

```diff
@@ -38,14 +38,14 @@
 import org.apache.iceberg.jdbc.UncheckedInterruptedException;
 import org.apache.iceberg.jdbc.UncheckedSQLException;
 import org.assertj.core.api.Assertions;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.ArgumentMatchers;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
```

Comment:
Can you explain why the lenient mode needs to be set?

Also you can set the strictness mode at the class level: `@MockitoSettings(strictness = Strictness.LENIENT)`